ml-agents/mlagents/trainers/bc/offline_trainer.py

```diff
@@ -15,17 +15,19 @@
 class OfflineBCTrainer(BCTrainer):
     """The OfflineBCTrainer is an implementation of Offline Behavioral Cloning."""
 
-    def __init__(self, brain, trainer_parameters, training, load, seed, run_id):
+    def __init__(self, brain, trainer_parameters, training, load, seed,
+                 run_id, debug_flag):
         """
         Responsible for collecting experiences and training PPO model.
         :param  trainer_parameters: The parameters for the trainer (dictionary).
         :param training: Whether the trainer is set for training.
         :param load: Whether the model should be loaded.
         :param seed: The seed the model will be initialized with
         :param run_id: The The identifier of the current run
+        :param debug_flag: Log debug statements
         """
         super(OfflineBCTrainer, self).__init__(
-            brain, trainer_parameters, training, load, seed, run_id)
+            brain, trainer_parameters, training, load, seed, run_id, debug_flag)
```

Comment:
Any thoughts on why this should be a debug print statement whereas the logging every N steps is not?  Relevant to the whole discussion we had about whether to print at `summary_freq` at all.  Curious how you came to this decision.

Reply:
We discussed a bit more and decided to keep the current functionality as is and move this to be a debug print b/c it wasn't clear whether this was actually useful to the person monitoring the command line.

Reply:
Sounds good